babel.config.js

```diff
@@ -30,7 +30,9 @@ module.exports = {
         loose: true
       }
     ],
-    'babel-plugin-styled-components'
+    'babel-plugin-styled-components',
+    ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
+    ['@babel/plugin-proposal-private-methods', { loose: true }]
```

Comment:
These plugins need to also use `loose` mode. This mode helps reduce bundle sizes. More details can be found [here](https://github.com/zendeskgarden/react-containers/pull/162).